helm/vitess/templates/_vtctld.tpl: Removal of web_dir flags can break UI for older binaries

Minikube version: v0.33.1
Helm version:
```
Client: &version.Version{SemVer:"v2.16.1", GitCommit:"bbdfe5e7803a12bbdf97e94cd847859890cf4050", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.16.1", GitCommit:"bbdfe5e7803a12bbdf97e94cd847859890cf4050", GitTreeState:"clean"}
```

I ran into a few issues while trying to run through the Kubernetes tutorial:

1. When running `kvtctld.sh` (after making it executable, see #5719), my browser tries to open `http://192.168.39.142:31333/app2/` and I get a `404 page not found` error. 

2. Figuring this was related to #5597, I tried changing the images from `helm-1.0.7-5` -> `latest`, though after that all of my tablets were `CrashLoopBackOff`ing

Here are the vttablet + mysql logs:
**mysql**
```
+ '[' mysql56 = percona ']'
+ '[' mysql56 = mysql ']'
+ '[' mysql56 = mysql56 ']'
+ MYSQL_FLAVOR=MySQL56
+ export MYSQL_FLAVOR
++ cat
+++ cat /vtdataroot/tabletdata/tablet-uid
+ eval exec /vt/bin/mysqlctld -logtostderr=true -stderrthreshold=0 -tablet_dir '"tabletdata"' -tablet_uid '"794219800"' -socket_file '"/vtdataroot/mysqlctl.sock"' -init_db_sql_file '"/vt/config/init_db.sql"'
++ exec /vt/bin/mysqlctld -logtostderr=true -stderrthreshold=0 -tablet_dir tabletdata -tablet_uid 794219800 -socket_file /vtdataroot/mysqlctl.sock -init_db_sql_file /vt/config/init_db.sql
ERROR: logging before flag.Parse: E0116 01:00:01.306804       1 syslogger.go:122] can't connect to syslog
I0116 01:00:01.313152       1 mysqlctld.go:75] mycnf file (/vtdataroot/tabletdata/my.cnf) doesn't exist, initializing
I0116 01:00:01.313346       1 dbconfigs.go:303] DBConfigs: {
  "Conn": {
    "host": "",
    "port": 0,
    "uname": "vt_dba",
    "pass": "****",
    "dbname": "",
    "unix_socket": "/vtdataroot/tabletdata/mysql.sock",
    "charset": "utf8",
    "flags": 0,
    "ssl_ca": "",
    "ssl_ca_path": "",
    "ssl_cert": "",
    "ssl_key": "",
    "server_name": "",
    "DeprecatedDBName": "",
    "DisableClientDeprecateEOF": false+ '[' mysql56 = percona ']'
+ '[' mysql56 = mysql ']'
+ '[' mysql56 = mysql56 ']'
+ MYSQL_FLAVOR=MySQL56
+ export MYSQL_FLAVOR
++ cat
+++ cat /vtdataroot/tabletdata/tablet-uid
+++ hostname
+ eval exec /vt/bin/vttablet '-topo_implementation="etcd2"' '-topo_global_server_address="etcd-global-client.default:2379"' -topo_global_root=/vitess/global -logtostderr -port 15002 -grpc_port 16002 -service_map '"grpc-queryservice,grpc-tabletmanager,grpc-updatestream"' -tablet_dir '"tabletdata"' -tablet-path '"zone1-794219800"' -tablet_hostname '"zone1-commerce-0-rdonly-0.vttablet"' -init_keyspace '"commerce"' -init_shard '"0"' -init_tablet_type '"rdonly"' -health_check_interval '"5s"' -mysqlctl_socket '"/vtdataroot/mysqlctl.sock"' -enable_replication_reporter -init_db_name_override '"commerce"' '-app_pool_size="40"' '-dba_pool_size="20"' '-queryserver-config-max-result-size="10000"' '-queryserver-config-pool-size="24"' '-queryserver-config-query-timeout="30"' '-queryserver-config-stream-pool-size="100"' '-queryserver-config-transaction-cap="300"'
++ exec /vt/bin/vttablet -topo_implementation=etcd2 -topo_global_server_address=etcd-global-client.default:2379 -topo_global_root=/vitess/global -logtostderr -port 15002 -grpc_port 16002 -service_map grpc-queryservice,grpc-tabletmanager,grpc-updatestream -tablet_dir tabletdata -tablet-path zone1-794219800 -tablet_hostname zone1-commerce-0-rdonly-0.vttablet -init_keyspace commerce -init_shard 0 -init_tablet_type rdonly -health_check_interval 5s -mysqlctl_socket /vtdataroot/mysqlctl.sock -enable_replication_reporter -init_db_name_override commerce -app_pool_size=40 -dba_pool_size=20 -queryserver-config-max-result-size=10000 -queryserver-config-pool-size=24 -queryserver-config-query-timeout=30 -queryserver-config-stream-pool-size=100 -queryserver-config-transaction-cap=300
ERROR: logging before flag.Parse: E0116 01:00:01.532860       1 syslogger.go:122] can't connect to syslog
I0116 01:00:01.547362       1 streamlog.go:155] Streaming logs from TabletServer at /debug/querylog.
I0116 01:00:01.547384       1 streamlog.go:155] Streaming logs from TxLog at /debug/txlog.
I0116 01:00:01.547423       1 trace.go:151] successfully started tracing with [noop]
I0116 01:00:01.547510       1 mycnf_flag.go:123] No mycnf_server_id, no mycnf-file specified, using default config for server id 794219800: /vtdataroot/tabletdata/my.cnf
F0116 01:00:01.547538       1 vttablet.go:80] mycnf read failed: open /vtdataroot/tabletdata/my.cnf: no such file or directory
  },
  "Users": {
    "dba": "vt_dba"
  }
}
I0116 01:00:01.313582       1 mysqld.go:586] execCmd: /usr/sbin/mysqld /usr/sbin/mysqld [--version]
I0116 01:00:01.317302       1 mysqld.go:600] execCmd: /usr/sbin/mysqld output: /usr/sbin/mysqld  Ver 5.7.26-29 for Linux on x86_64 (Percona Server (GPL), Release 29, Revision 11ad961)
I0116 01:00:01.317342       1 mysqld.go:174] Using flavor: percona, version: {5 7 26}
I0116 01:00:01.317356       1 mysqld.go:639] mysqlctl.Init
I0116 01:00:01.317358       1 mysqld.go:621] mysqlctl.InitConfig
I0116 01:00:01.317362       1 mysqld.go:915] creating directory /vtdataroot/tabletdata
I0116 01:00:01.317378       1 mysqld.go:949] creating directory /vtdataroot/tabletdata/data
I0116 01:00:01.317385       1 mysqld.go:949] creating directory /vtdataroot/tabletdata/innodb
I0116 01:00:01.317390       1 mysqld.go:949] creating directory /vtdataroot/tabletdata/relay-logs
I0116 01:00:01.317396       1 mysqld.go:949] creating directory /vtdataroot/tabletdata/bin-logs
I0116 01:00:01.317401       1 mysqld.go:925] creating directory /vtdataroot/tabletdata/data
I0116 01:00:01.317404       1 mysqld.go:925] creating directory /vtdataroot/tabletdata/innodb/data
I0116 01:00:01.317408       1 mysqld.go:925] creating directory /vtdataroot/tabletdata/innodb/logs
I0116 01:00:01.317412       1 mysqld.go:925] creating directory /vtdataroot/tabletdata/tmp
I0116 01:00:01.317417       1 mysqld.go:925] creating directory /vtdataroot/tabletdata/relay-logs
I0116 01:00:01.317430       1 mysqld.go:925] creating directory /vtdataroot/tabletdata/bin-logs
I0116 01:00:01.317442       1 mysqld.go:781] make_mycnf hook doesn't exist, reading template files
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xab6238]
goroutine 1 [running]:
vitess.io/vitess/go/exit.doRecover(0xc52120, 0x16978c0, 0x1697800)
	/vt/src/vitess.io/vitess/go/exit/exit.go:92 +0x1b8
vitess.io/vitess/go/exit.Recover()
	/vt/src/vitess.io/vitess/go/exit/exit.go:69 +0x48
panic(0xc52120, 0x16978c0)
	/usr/local/go/src/runtime/panic.go:522 +0x1b5
github.com/GeertJohan/go%2erice.(*Box).IsEmbedded(...)
	/go/pkg/mod/github.com/!geert!johan/go.rice@v1.0.0/box.go:157
github.com/GeertJohan/go%2erice.(*Box).Open(0x0, 0xd8050f, 0x11, 0xc000399a08, 0x4f1262, 0xc00011e680)
	/go/pkg/mod/github.com/!geert!johan/go.rice@v1.0.0/box.go:185 +0x48
github.com/GeertJohan/go%2erice.(*Box).Bytes(0x0, 0xd8050f, 0x11, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/!geert!johan/go.rice@v1.0.0/box.go:280 +0x5c
vitess.io/vitess/go/vt/mysqlctl.(*Mysqld).getMycnfTemplate(0xc000131e30, 0x35, 0x0)
	/vt/src/vitess.io/vitess/go/vt/mysqlctl/mysqld.go:800 +0x9c
vitess.io/vitess/go/vt/mysqlctl.(*Mysqld).initConfig(0xc000131e30, 0xc0000f57a0, 0xc0000c2460, 0x1d, 0x0, 0xd7cea0)
	/vt/src/vitess.io/vitess/go/vt/mysqlctl/mysqld.go:782 +0x40c
vitess.io/vitess/go/vt/mysqlctl.(*Mysqld).InitConfig(0xc000131e30, 0xc0000f57a0, 0x0, 0x0)
	/vt/src/vitess.io/vitess/go/vt/mysqlctl/mysqld.go:628 +0x17f
vitess.io/vitess/go/vt/mysqlctl.(*Mysqld).Init(0xc000131e30, 0xeafe20, 0xc0000c6960, 0xc0000f57a0, 0x7ffe960b05ef, 0x16, 0x0, 0x0)
	/vt/src/vitess.io/vitess/go/vt/mysqlctl/mysqld.go:640 +0x89
main.main()
	/vt/src/vitess.io/vitess/go/cmd/mysqlctld/mysqlctld.go:85 +0x3d2
```

**vttablet**
```
+ '[' mysql56 = percona ']'
+ '[' mysql56 = mysql ']'
+ '[' mysql56 = mysql56 ']'
+ MYSQL_FLAVOR=MySQL56
+ export MYSQL_FLAVOR
++ cat
+++ cat /vtdataroot/tabletdata/tablet-uid
+++ hostname
+ eval exec /vt/bin/vttablet '-topo_implementation="etcd2"' '-topo_global_server_address="etcd-global-client.default:2379"' -topo_global_root=/vitess/global -logtostderr -port 15002 -grpc_port 16002 -service_map '"grpc-queryservice,grpc-tabletmanager,grpc-updatestream"' -tablet_dir '"tabletdata"' -tablet-path '"zone1-794219800"' -tablet_hostname '"zone1-commerce-0-rdonly-0.vttablet"' -init_keyspace '"commerce"' -init_shard '"0"' -init_tablet_type '"rdonly"' -health_check_interval '"5s"' -mysqlctl_socket '"/vtdataroot/mysqlctl.sock"' -enable_replication_reporter -init_db_name_override '"commerce"' '-app_pool_size="40"' '-dba_pool_size="20"' '-queryserver-config-max-result-size="10000"' '-queryserver-config-pool-size="24"' '-queryserver-config-query-timeout="30"' '-queryserver-config-stream-pool-size="100"' '-queryserver-config-transaction-cap="300"'
++ exec /vt/bin/vttablet -topo_implementation=etcd2 -topo_global_server_address=etcd-global-client.default:2379 -topo_global_root=/vitess/global -logtostderr -port 15002 -grpc_port 16002 -service_map grpc-queryservice,grpc-tabletmanager,grpc-updatestream -tablet_dir tabletdata -tablet-path zone1-794219800 -tablet_hostname zone1-commerce-0-rdonly-0.vttablet -init_keyspace commerce -init_shard 0 -init_tablet_type rdonly -health_check_interval 5s -mysqlctl_socket /vtdataroot/mysqlctl.sock -enable_replication_reporter -init_db_name_override commerce -app_pool_size=40 -dba_pool_size=20 -queryserver-config-max-result-size=10000 -queryserver-config-pool-size=24 -queryserver-config-query-timeout=30 -queryserver-config-stream-pool-size=100 -queryserver-config-transaction-cap=300
ERROR: logging before flag.Parse: E0116 01:00:01.532860       1 syslogger.go:122] can't connect to syslog
I0116 01:00:01.547362       1 streamlog.go:155] Streaming logs from TabletServer at /debug/querylog.
I0116 01:00:01.547384       1 streamlog.go:155] Streaming logs from TxLog at /debug/txlog.
I0116 01:00:01.547423       1 trace.go:151] successfully started tracing with [noop]
I0116 01:00:01.547510       1 mycnf_flag.go:123] No mycnf_server_id, no mycnf-file specified, using default config for server id 794219800: /vtdataroot/tabletdata/my.cnf
F0116 01:00:01.547538       1 vttablet.go:80] mycnf read failed: open /vtdataroot/tabletdata/my.cnf: no such file or directory
```

--- a/helm/vitess/templates/_vtctld.tpl
+++ b/helm/vitess/templates/_vtctld.tpl
@@ -90,6 +90,8 @@ spec:
 
               eval exec /vt/bin/vtctld $(cat <<END_OF_COMMAND
                 -cell={{$cellClean | quote}}
+                -web_dir="/vt/web/vtctld"
+                -web_dir2="/vt/web/vtctld2/app"
                 -workflow_manager_init
                 -workflow_manager_use_election
                 -logtostderr=true

